test/README.md

```diff
@@ -0,0 +1,22 @@
+Scylla in-source tests.
+
+For details on how to run the tests, see docs/dev/testing.md
+
+Shared C++ utils, libraries are in lib/, for Pythoin - pylib/
+
+alternator - client/server tests for Scylla Alternator
+unit, boost, raft - unit tests in C++
+cql-pytest - Python tests which connect to a single server and use CQL
+topology* - tests that set up clusters and add/remove nodes
+cql - approval tests that use CQL and pre-recorded output
+rest\_api - tests for Scylla REST API Port 9000
+
```

Comment:
missing: nodetool directory

Reply:
also scylla-gdb. And there are more. Too many ;-)

Reply:
and resource/. It took me a while to understand that there are pre-generated (rather than run-time-generated) sstables on which some unit tests are run

Reply:
Yes. In general, I think it's a mistake that we split tests across directories - like a unit test in boost/ that needs files from resource/, or a test in topology/ which uses a library from pylib/. I think a proper organization would be to put the tests next to everything they need. But we don't need to reorganize things in this patch. But we do need to explain the current organization.

Reply:
I added the two above, but there is no goal to list them all. The list will quickly become obsolete.